# Normal Tolerance Interval Example

In [1]:
import normtolint as nti

Example 4.8 from Meeker, William Q.; Hahn, Gerald J.; Escobar, Luis A.. Statistical Intervals: A Guide for Practitioners and Researchers (Wiley Series in Probability and Statistics) (p. 54). Wiley. Kindle Edition.

Assume an electronic circuit is designed to produce an output voltage. For $n = 5$ units, the voltages show mean $\bar{x} = 50.10$ volts and standard deviation $s = 1.31$ volts. Suppose now that the manufacturer wanted a two-sided 95% confidence tolerance interval to contain a proportion 0.90 of the distribution of the broader population of shipping units under a normal distribution assumption.

In [2]:
n = 5
x_bar = 50.1
s = 1.31
coverage = 0.90
confidence = 0.95

Compute the appropriate tolerance factor.

In [3]:
k = nti.tolerance_factor(n, coverage, confidence)

Form the interval.

In [4]:
(x_bar - k * s, x_bar + k * s)

(44.47930866740012, 55.720691332599884)

The text shows [44.5, 55.7] as the answer for this example.